bindings/pyfiction/include/pyfiction/technology/charge_distribution_surface.hpp

```diff
@@ -294,6 +293,17 @@ inline void charge_distribution_surfaces(pybind11::module& m)
 
         ;
 
+    /**
+     * Assign charge state mode.
+     */
+    py::enum_<fiction::charge_index_mode>(m, "charge_index_mode", DOC(fiction_energy_calculation), py::module_local())
+        .value("KEEP_OLD_ENERGY_VALUE", fiction::charge_index_mode::UPDATE_CHARGE_INDEX,
+               DOC(fiction_charge_index_mode_UPDATE_CHARGE_INDEX))
+        .value("UPDATE_ENERGY", fiction::charge_index_mode::DO_NOT_UPDATE_CHARGE_INDEX,
+               DOC(fiction_charge_index_mode_DO_NOT_UPDATE_CHARGE_INDEX))
+
+        ;
```

Comment:
Could you add a Python test involving this `enum`?

Reply:
sure, good point!